extensions/spring-data-jpa/deployment/src/test/resources/import_customers.sql

```diff
@@ -1,11 +1,11 @@
-INSERT INTO "Customer$Country" (id, name, isocode)
+INSERT INTO Customer$Country (id, name, isocode)
```

Comment:
Could you explain why this change?

Asking because it could impact users.

Reply:
It's discussed in (too) great details here: https://hibernate.zulipchat.com/#narrow/channel/132094-hibernate-orm-dev/topic/Uppercase.20table.20names.20with.20H2.20in.20ORM.207.3F/with/506173896

Bottom line: it's a behavior change to fix a regression. 5.x behaved a certain way, 6.x another, and in 7.x we're back to the 5.x behavior.

I agree it'd worth mentioning in the migration guide.